src/Modules/UngradedModules/FreeMod.jl

```diff
@@ -49,23 +49,23 @@ julia> U = complement_of_prime_ideal(P);
 julia> RL, _ = localization(R, U);
 
 julia> FRL = free_module(RL, 2, "f")
-Free module of rank 2 over Localization of multivariate polynomial ring in 3 variables over QQ at complement of prime ideal(x, y, z)
+Free module of rank 2 over localized ring
```

Comment:
maybe `localized polynomial ring` would be better, given that we also spell out `multivariate polynomial ring`? But I have no strong opinion on that .

Reply:
Maybe, but that's beyond the scope of this PR, the `show` method for `AbsLocalizedRing` would have to be adjusted.